Dockerfile.QA

```diff
@@ -44,19 +44,34 @@ FROM ${BUILD_IMAGE} AS build
 ENV DEBIAN_FRONTEND=noninteractive
 
 # image_preprocess custom backend requires opencv
+# go client requires golang
 RUN apt-get update && apt-get install -y --no-install-recommends \
             libopencv-dev \
-            libopencv-core-dev
+            libopencv-core-dev \
+            golang-go \
+            protobuf-compiler
+
+apt-get update && apt-get install -y --no-install-recommends \
+            golang-go \
+            protobuf-compiler
 
 WORKDIR /workspace
 RUN cd builddir && \
     make -j16 test-util custom-backend
 
+# need protoc-gen-go to generate go specific gRPC modules
+RUN go get github.com/golang/protobuf/protoc-gen-go && \
```

Comment:
Put this at line 53 after the apt-get run. This is part of setting up packages so it should stay together with similar commands.